### PR TITLE
Add section on cluster routing capability

### PR DESCRIPTION
**Pull request by gjmwoods**

### CODE REVIEW

File: modules/ROOT/pages/actions/routing.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+:description: HTTP API Routing.
+
+[[http-api-actions-cluster-routing]]
+= Clusters and Routing
+
+The Cypher Transactional API provides a limited level of support for query routing. The level of support depends on whether Server-side Routing has been enabled. See (`dbms.routing.enabled=true` [link to https://neo4j.com/docs/operations-manual/current/clustering/internals/#clustering-routing)].
```
Comment (AlexicaWright) [inline modules/ROOT/pages/actions/routing.adoc]: New sentence = new line

File: modules/ROOT/pages/actions/routing.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+== Server-side Routing Disabled
+
+Clusters support read-only queries.
+
+== Server-side Routing Enabled
+
+Clusters support routing for queries that involve a xref::/actions/begin-and-commit-a-transaction-in-one-request.adoc[single request] (i.e. via the `tx/commit` endpoint). This is because the cluster does not currently support transaction identifiers across the cluster.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/actions/routing.adoc]: Same here, new sentence = new line

File: modules/ROOT/pages/actions/routing.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+In this context, all queries are considered `WRITE` queries even if they contain no cypher with write operations. Since the HTTP API did not previously support marking queries as `READ` or `WRITE`, the server assumes all queries are `WRITE` queries to support all cases. The effect of this is that all queries are ultimately executed on the leader.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/actions/routing.adoc]: As above

**Review by AlexicaWright — COMMENTED** [review]
Just editorial suggestions. LGTM otherwise 👍

File: modules/ROOT/pages/actions/routing.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
+:description: HTTP API Routing.
+
+[[http-api-actions-cluster-routing]]
+= Clusters and Routing
+
+The Cypher Transactional API provides a limited level of support for query routing.
+The level of support depends on whether xref:4.4-preview@operations-manual:ROOT:clustering/internals.adoc#causal-clustering-routing[Server-side Routing]  has been enabled.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/actions/routing.adoc:7]:
```suggestion
The level of support depends on whether link:https://neo4j.com/docs/operations-manual/current/clustering/internals/#clustering-routing[Server-side Routing] has been enabled.
```

Comment (AlexicaWright) [reply modules/ROOT/pages/actions/routing.adoc:7]: This is how we link elsewhere between the manuals. Also, using `4.4-preview` will probably not work once 4.4 is released.

**Comment by AlexicaWright** [comment]
👍 LGTM!

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @gjmwoods & @AlexicaWright 

I pushed an editorial change to change the page from sections to a table: 
Screenshot 2021-11-11 at 16 19 01

This seemed cleaner to me, but please let me know if this is not an improvement!

**Pull request merged**